mappings/net/minecraft/entity/EntityWithAiGoals.mapping

```diff
@@ -1,4 +1,17 @@
 CLASS us net/minecraft/entity/EntityWithAiGoals
-	FIELD a pos Ldx;
+	FIELD a aiCenter Ldx;
```

Comment:
I'll have to look at the code more closely to decide if it's a real home or not. My first interpretation was that it's not, but it's worth looking into.